omero/users/cli/chgrp.txt

```diff
@@ -0,0 +1,140 @@
+
+    $ bin/omero chgrp 5 OriginalFile:101
+    $ bin/omero chgrp Group:5 Project:51
+    $ bin/omero chgrp ExperimenterGroup:5 Project:51
+    $ bin/omero chgrp lab_group Project:51
+
+In the first line, the original file with ID 101 will be moved to the group
+with ID 5. In the second, project 51 will be moved to group 5 including any
+datasets inside only that project and any images that are only contained within
+any moved datasets. If group 5 is named 'lab_group' then the last two lines
+will have the same effect as the second.
+
+Moving multiple objects
+=======================
+
+Multiple objects can be specified with each type being followed by an ID
+or a comma-separated list of IDs. The order of objects or IDs is not
+significant, thus all three calls below are identical in moving
+project 51 and datasets 53 and 54 to group 5.::
+
+    $ bin/omero chgrp 5 Project:51 Dataset:53,54
+    $ bin/omero chgrp 5 Dataset:54,53 Project:51
+    $ bin/omero chgrp 5 Dataset:53 Project:51 Dataset:54
+
+Moving lower level objects
+==========================
+
+To move objects below a specified top-level object the following form
+of the object specifier is used.::
+
+    $ bin/omero chgrp 5 Project/Dataset/Image:51
+
+Here the all of images under the project 51 would be moved. It is
+necessary to specify intermediate objects in the hierarchy and so::
+
+    $ bin/omero chgrp 5 Project/Image:51
+
+would have the same effect as the call above. Links can also be moved
+and so::
+
+$ bin/omero chgrp 5 Project/DatasetImageLink:51 Dataset/DatasetImageLink:53
+
+would effectively orphan all images under project 51 and dataset 53 that are
+not also under other datasets.
+
+Including and excluding objects
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. option:: --include
+
+    Linked objects that would not ordinarily be moved can be included in the
+    move using the `--include` option.::
+
+        $ bin/omero chgrp 5 Image:51 --include Annotation
+
+    This call would move any annotation objects linked to the image.
+
+.. option:: --exclude
+
+    Linked objects that would ordinarily be moved can be excluded from the
+    move using the `--exclude` option.::
+
+        $ bin/omero chgrp 5 Project:51 --exclude Dataset
+
+    This will move project 51 but not any datasets contained in that project.
+
+The two options can be used together.::
+
+     $ bin/omero chgrp 5 Project/Dataset:53 --exclude Image --include:FileAnnotation
+
+This will move any datasets under project 53, that are not otherwise
+contained elsewhere, excluding any images in those datasets but including
+and file annotations linked to the moved datasets. In this case the images
+that are not otherwise contained in datasets will be orphaned.
+
+Further options
+^^^^^^^^^^^^^^^
+
+.. option:: --ordered
+
+    Move the objects in the order specified.
+
+    Normally all of the specified objects are grouped into a single move
+    command if possible. However, each object can be moved separately and
```

Comment:
don't need "if possible" unless we know of any exceptions, "normally" is already weakening